packages/algob/sample-project/algobpy/parse.py

```diff
@@ -0,0 +1,17 @@
+import sys
+import yaml
+
+'''
+Overwrites scParam values if args is defined and has keys common from scParam
+'''
+def parseArgs(args, scParam):
```

Comment:
Please rename this function to `parse_params`

Reply:
done